src/shared/codelens/localLambdaRunner.ts

```diff
@@ -55,6 +55,12 @@ export interface OnDidSamBuildParams {
     isDebug: boolean
 }
 
+export interface DebuggerMetadata {
+    success: boolean
+    duration?: number
```

Comment:
In what situation will `duration` or `attempts` be undefined?

Reply:
I think the idea was that failures wouldn't produce this data, which I'm now thinking isn't a great idea.